grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy

```diff
@@ -101,11 +101,13 @@ class InvoiceService {
             invoiceItems = InvoiceItem.createCriteria().list(max: max.toInteger(), offset: offset.toInteger()) {
                 eq("invoice", invoice)
                 order("inverse", "asc")
+                order("id", "asc")
```

Comment:
Perhaps sorting by product code here would be better?

Reply:
No, we can invoice a few products with the same product code and the same issue will be visible.